docs/release-notes.d/swap-using-offset.md

```diff
@@ -0,0 +1,8 @@
+- Added a new swap using offset algorithm which is set with
+  `MCUBOOT_SWAP_USING_OFFSET`. This algorithm is similar to swap
+  using move but avoids moving the sectors in the primary slot
+  up by having the update image written in the second sector in
+  the update slot, which offers a faster update process and
+  requires a smaller swap status area
+- Made swap using offset the default algorithm for Zephyr builds
```

Comment:
This is set as default if preferred... as it was with the move alg before

Reply:
Removed this line as it is not being made the default, can submit another PR after if we want to make it default